### PR TITLE
CSS TASK

### PR DESCRIPTION
**Pull request by kolawolejohn**
This file contains my CSS and modifications to my HTML file, to suit the CSS additions.
- login.css

### CODE REVIEW

File: submissions/kolawoleOgunfowokan/HTML/css/login.css [hunk 1]
```diff
@@ -0,0 +1,109 @@
+*{
+    box-sizing: border-box;
+}
+body{
```
Comment (ToluAkin) [inline submissions/kolawoleOgunfowokan/HTML/css/login.css]: There should be a space between the attribute and the opening curly bracket. 
Also, there should be a space between each closing  bracket and a new attribute declaration.
This is for proper indentation.

File: submissions/kolawoleOgunfowokan/HTML/css/login.css [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    /*background-color: #0919f7;*/
```
Comment (ToluAkin) [inline submissions/kolawoleOgunfowokan/HTML/css/login.css]: The background-color in the comment is not the same as the one declared. Please adjust it accordingly.

File: submissions/kolawoleOgunfowokan/HTML/css/login.css (hunk 1 above)
Comment (kolawolejohn) [reply submissions/kolawoleOgunfowokan/HTML/css/login.css]: Done

File: submissions/kolawoleOgunfowokan/HTML/css/login.css (hunk 2 above)
Comment (kolawolejohn) [reply submissions/kolawoleOgunfowokan/HTML/css/login.css]: Done

File: submissions/kolawoleOgunfowokan/HTML/css/style.css [hunk 3]
```diff
@@ -0,0 +1,140 @@
+* {
+
```
Comment (ToluAkin) [inline submissions/kolawoleOgunfowokan/HTML/css/style.css]: Please remove this space everywhere else. It is not needed.

Comment (kolawolejohn) [reply submissions/kolawoleOgunfowokan/HTML/css/style.css]: Done, the spaces have been removed.

**Comment by ToluAkin** [comment]
Well done @kolawolejohn

**Review by ToluAkin — APPROVED** [review]

**Review by goke-epapa — APPROVED** [review]
Nice work, well done

**Pull request merged**